release/example/en.custom/source/wordbreak.ts

```diff
@@ -1,18 +1,21 @@
 /*
   Naively treat any character that is not a letter as a wordbreaking character.
 */
-com.keyman.lexicalModel.registerWordBreaker('example.en.custom', function(text: string): Array<number> {
-  let regex = new RegExp(/[!a-zA-Z]/g);
-
-  let indexesOf = function(string) {
-    var match, indexes = [];
-
-    while (match = regex.exec(string)) {
-      indexes.push(match.index);
+class ExampleWordBreaker {
+  // TODO:  Needs to reference a .d.ts file for type definitions!
+  break(text: string): string[] {
+    let regex = new RegExp(/[!a-zA-Z]/g);
+  
+    let tokensOf = function(string: string) {
+      var match: RegExpExecArray, tokens: string[] = [];
+  
+      while (match = regex.exec(string)) {
+        tokens.push(match[0]);
+      }
+  
+      return tokens;
     }
-
-    return indexes;
+  
+    return tokensOf(text);
```

Comment:
There are a few ways that we could design this API. One is to return the start indices (as before), and another is to return the tokens themselves. Another way is to return both start and end indicies, AND the token token text. One of these attributes could even be an accessor rather than a "real" attribute of the class, e.g.,

```typescript
class WordBreak {
  start: number;
  end: number;
  private source: string;
  get token() {
     return this.source.substr(this.start, this.end);
  }
}
```

This does **not** need to be addressed in this PR, but I'm just putting it out there. I still don't know what interface is the most convenient for implementors.

 Does not need to be addressed in

Reply:
Yeah, I'm not exactly sure on which way to go.  Tokenization is probably the easiest to consume for lexical models - hence why I chose it - though it might get in the way when the time comes to build a new `Transform` to apply for a suggestion, since it doesn't supply indices.

Worst case, there's always the "why not both?" option.  But yeah, that can be addressed in followup work; we've yet to implement a default word-breaker, after all.